src/Development/IDE/LSP/LanguageServer.hs

```diff
@@ -133,7 +133,7 @@ runLanguageServer options userHandlers onInitialConfig onConfigChange getIdeStat
         handleInit exitClientMsg clearReqId waitForCancel clientMsgChan lspFuncs@LSP.LspFuncs{..} = do
 
             ide <- getIdeState getNextReqId sendFunc (makeLSPVFSHandle lspFuncs) clientCapabilities
-                               withProgress withIndefiniteProgress
+                               withProgress withIndefiniteProgress lspFuncs
```

Comment:
A bunch of the arguments to `getIdeState` are contained inside `lspFuncs`. Please simplify

Reply:
Didn't realize. Fixed.